### PR TITLE
test(@desktop/general): can quit app by going to settings -> quit

### PR DESCRIPTION
**Pull request by saledjenic**
Fixes: #6905

### CODE REVIEW

**Review by caybro — APPROVED** [review]
Code looks good

**Review by noeliaSD — APPROVED** [review]
Nice job and tested!

Just only wondering, as I told you, if the location of `process_terminated` and `verify_the_app_is_closed` should be in `SquishDriverVerification`.

The propose of `SquishDriverVerification` is just to add wrapper methods of the squish `test.XX` api and / or common and global validation methods as tools but not to add specific validations there.

In case of:
- `verify_the_app_is_closed`:  I see it as a direct method of `SettingsScreen` since it is a specific validation for closing the app. Not a tool that can be reused in other places.
- `process_terminated`:  I see it as a util, like methods from `FileManager`. Maybe we could create a new `Utils` or `ProcessesManager` file.

WDYT?

File: test/ui-test/testSuites/suite_status/tst_settingsMenu/test.feature [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    As a user I want to login the app go to Settings and go through all settings menu
+    checking the result of each menu item
+
+    Background: Sign up and open settings section
+        Given A first time user lands on the status desktop and generates new key
+        When user signs up with username tester123 and password TesTEr16843/!@00
+        Then the user lands on the signed in app
+        When the user opens app settings screen
+
+    Scenario: User quits the app
```
Comment (noeliaSD) [inline test/ui-test/testSuites/suite_status/tst_settingsMenu/test.feature:12]: Agreed on using `the user` insted of `user`
```suggestion
    Scenario: The user quits the app
```

File: test/ui-test/testSuites/suite_status/tst_settingsMenu/test.feature [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+		When user clicks on Sign out and Quit
```
Comment (noeliaSD) [inline test/ui-test/testSuites/suite_status/tst_settingsMenu/test.feature:13]: Agreed on using `the user` insted of `user`
```suggestion
		When the user clicks on Sign out and Quit
```

**Comment by saledjenic** [comment]
> Just only wondering, as I told you, if the location of process_terminated and verify_the_app_is_closed should be in SquishDriverVerification.

@noeliaSD they are already in `SquishDriverVerification`, not sure what you mean?

> verify_the_app_is_closed: I see it as a direct method of SettingsScreen since it is a specific validation for closing the app. Not a tool that can be reused in other places.

@noeliaSD at the moment it is used by the `SettingsScreen`, but we have few more ways to close the app, not only from the main menu of settings section, so I see it as a shared verification function

> process_terminated: I see it as a util, like methods from FileManager. Maybe we could create a new Utils or ProcessesManager file.

@noeliaSD I really doubt that this function will be ever used directly, actually it should not, it's more a private function in `SquishDriverVerification` and used only by the `verify_the_app_is_closed` function. So whenever we want to check if the app is closed we should do that calling `verify_the_app_is_closed` function.

**Comment by noeliaSD** [comment]
> > Just only wondering, as I told you, if the location of process_terminated and verify_the_app_is_closed should be in SquishDriverVerification.
> 
> @noeliaSD they are already in `SquishDriverVerification`, not sure what you mean?
> 
> > verify_the_app_is_closed: I see it as a direct method of SettingsScreen since it is a specific validation for closing the app. Not a tool that can be reused in other places.
> 
> @noeliaSD at the moment it is used by the `SettingsScreen`, but we have few more ways to close the app, not only from the main menu of settings section, so I see it as a shared verification function
> 
> > process_terminated: I see it as a util, like methods from FileManager. Maybe we could create a new Utils or ProcessesManager file.
> 
> @noeliaSD I really doubt that this function will be ever used directly, actually it should not, it's more a private function in `SquishDriverVerification` and used only by the `verify_the_app_is_closed` function. So whenever we want to check if the app is closed we should do that calling `verify_the_app_is_closed` function.

> @noeliaSD they are already in `SquishDriverVerification`, not sure what you mean?

I mean I'm not sure of having it here (`SquishDriverVerification`) as explained in the previous bullet points for each method. 

Ok, let's keep it there but still doubting on it, as said before, `SquishDriverVerification` main propose is for adding validation utils methods not to define specific validators as it is done in this case.

**Pull request merged**